backend/database/__init__.py

```diff
@@ -24,35 +25,38 @@ def drop_all_tables(db: sqlalchemy.engine.Connection) -> None:
     ))
 
 def create_all_tables(db: sqlalchemy.engine.Connection) -> None:
+    db.execute(text('''DROP TABLE IF EXISTS visitors'''))
```

Comment:
Optional: 
As per my prior comment on Emily's PR (I'm assuming you rebased off of Emily's PR), this line is unnecessary because there is already a drop_all_tables() function that is going to be called before this. Rather, I've split the functionality of creating and deleting of tables into two separate functions

(Not sure if this is better because) I wanted to keep to SRP if possible.

Reply:
noted on this. will not modify for this PR. should resolve it on PR #7 instead